src/Symfony/Bundle/FrameworkBundle/Command/SecretsDecryptToLocalCommand.php

```diff
@@ -38,6 +38,7 @@ public function __construct(
     protected function configure(): void
     {
         $this
+            ->addOption('exit', 'e', InputOption::VALUE_NONE, 'Returns a non-zero exit code if any errors are encountered')
```

Comment:
-e is already taken so better define no alias

Reply:
done